code/part-one/blockchain.js

```diff
@@ -94,17 +97,54 @@ class Blockchain {
   }
 
   /*
-    Add a new block to the blockchain:
+    Add a new block to the blockchain based on pending transactions:
 
     Create function that:
       - Adds previousHash property to new block
       - Create hash (or mine block)
       - Adds block to the chain
+      - Creates new pending transaction for mining reward
+  */
+  minePendingTransactions(miningRewardAddress) {
+    let block = new Block(
+      Date.now(), this.pendingTransactions, this.getLatestBlock().hash);
+    block.mineBlock(this._difficulty);
+
+    console.log('Block has been mined.');
+    this.chain.push(block);
+    
+    this.pendingTransactions = [
+      /*
+        Mining rewards do not come from any specific address
+        and instead are awarded directly from the chain itself.
+      */
+      new Transaction(null, miningRewardAddress, this.miningReward)
+    ];
+  }
+
+  createTransaction(transaction) {
```

Comment:
`addTransaction`? 

The verb `create` is confusing to me since you have create the transaction separately.

Reply:
Changed to addTransaction